lib/src/map/options/enabled_gestures.dart

```diff
@@ -0,0 +1,254 @@
+    this.scrollWheelZoom = true,
+    this.twoFingerRotate = true,
+    this.ctrlDragRotate = true,
+  });
+
+  /// This constructor has no enabled gestures by default. Use this constructor
+  /// if you want have no gestures enabled or only some specific gestures.
+  ///
+  /// In case you want have most or all of the gestures enabled use the
+  /// [InteractiveFlags.all] constructor instead.
+  const EnabledGestures.none({
+    this.drag = false,
+    this.flingAnimation = false,
+    this.twoFingerMove = false,
+    this.twoFingerZoom = false,
+    this.doubleTapZoomIn = false,
+    this.doubleTapDragZoom = false,
+    this.scrollWheelZoom = false,
+    this.twoFingerRotate = false,
+    this.ctrlDragRotate = false,
+  });
+
+  /// This constructor supports bitfield operations on the static fields
+  /// from [InteractiveFlag].
+  factory EnabledGestures.bitfield(int flags) {
+    return EnabledGestures._(
+      drag: InteractiveFlag.hasFlag(flags, InteractiveFlag.drag),
+      flingAnimation:
+          InteractiveFlag.hasFlag(flags, InteractiveFlag.flingAnimation),
+      twoFingerMove:
+          InteractiveFlag.hasFlag(flags, InteractiveFlag.twoFingerMove),
+      twoFingerZoom:
+          InteractiveFlag.hasFlag(flags, InteractiveFlag.twoFingerZoom),
+      doubleTapZoomIn:
+          InteractiveFlag.hasFlag(flags, InteractiveFlag.doubleTapZoomIn),
+      doubleTapDragZoom:
+          InteractiveFlag.hasFlag(flags, InteractiveFlag.doubleTapDragZoom),
+      scrollWheelZoom:
+          InteractiveFlag.hasFlag(flags, InteractiveFlag.scrollWheelZoom),
+      twoFingerRotate:
+          InteractiveFlag.hasFlag(flags, InteractiveFlag.twoFingerRotate),
+      ctrlDragRotate:
+          InteractiveFlag.hasFlag(flags, InteractiveFlag.ctrlDragRotate),
+    );
+  }
+
+  /// Enable panning with a single finger or cursor
+  final bool drag;
+
+  /// Enable fling animation after panning if velocity is great enough.
+  final bool flingAnimation;
+
+  /// Enable panning with multiple fingers
+  final bool twoFingerMove;
+
+  /// Enable zooming with a multi-finger pinch gesture
+  final bool twoFingerZoom;
+
+  /// Enable rotation with two-finger twist gesture
+  final bool twoFingerRotate;
+
+  /// Enable zooming with a single-finger double tap gesture
+  final bool doubleTapZoomIn;
+
+  /// Enable zooming with a single-finger double-tap-drag gesture
+  ///
+  /// The associated [MapEventSource] is [MapEventSource.doubleTapHold].
+  final bool doubleTapDragZoom;
+
+  /// Enable zooming with a mouse scroll wheel
+  final bool scrollWheelZoom;
+
+  /// Enable rotation by pressing the CTRL key and dragging with the cursor
+  /// or finger.
+  final bool ctrlDragRotate;
+
+  /// Returns true of any gesture with more than one finger is enabled.
+  bool hasMultiFinger() => twoFingerMove || twoFingerZoom || twoFingerRotate;
+
+  /// Wither to change the value of some gestures. Returns a new
+  /// [EnabledGestures] object.
+  EnabledGestures withFlag({
```

Comment:
Should probably use the standard `copyWith` name for clarity and consistency.

Reply:
Agree.